sa/_db/migrations/20150826133823_SCTReceipts.sql

```diff
@@ -0,0 +1,21 @@
+
+-- +goose Up
```

Comment:
This migration needs to be renamed to have a timestamp of today (or more realistically tomorrow).